packages/cli/src/commands/export-dynamic-plugin/backend-embed-as-dependencies.ts

```diff
@@ -659,6 +655,7 @@ function checkWorkspacePackageVersion(
 
 export function customizeForDynamicUse(options: {
   embedded: ResolvedEmbedded[];
+  isYarnV1: boolean;
```

Comment:
Actually you'll need to maybe [update this](https://github.com/janus-idp/backstage-plugins/blob/main/packages/cli/src/commands/export-dynamic-plugin/frontend.ts#L113-L126) as part of this PR too